cmd/containerd/server/server.go

```diff
@@ -472,17 +469,6 @@ func (s *Server) Wait() {
 // of all plugins.
 func LoadPlugins(ctx context.Context, config *srvconfig.Config) ([]plugin.Registration, error) {
 	// load all plugins into containerd
-	path := config.PluginDir //nolint:staticcheck
-	if path == "" {
-		path = filepath.Join(config.Root, "plugins")
-	}
-	if count, err := dynamic.Load(path); err != nil {
```

Comment:
(Alternatively maybe we can keep this line and return an error if `count > 0`)

@fuweid